discord/commands/core.py

```diff
@@ -582,6 +582,17 @@ def qualified_name(self) -> str:
         else:
             return self.name
 
+    @property
+    def qualified_id(self) -> int:
+        """:class:`int`: Retrieves the fully qualified command ID.
+
+        This is the root parent ID. For example, in ``/one two three``
+        the qualified ID would return ``one.id``.
+        """
+        if getattr(self, "parent", None) is not None:
+            return self.parent.qualified_id
```

Comment:
Would this logic be better?```suggestion
        if self.id is None:
            return self.parent.qualified_id
```

Reply:
It's made with the chance that `self.parent` is None but i suppose that makes more sense for returning IDs

Reply:
> It's made with the chance that `self.parent` is None but i suppose that makes more sense for returning IDs

IMO if there's an ID set, we should use it